src/nvim/syntax.c

```diff
@@ -6758,16 +6758,27 @@ void set_hl_group(int id, HlAttrs attrs, Dict(highlight) *dict, int link_id)
     { NULL, -1, NIL },
   };
 
+  char hex_name[8];
+  char *name;
+
   for (int j = 0; cattrs[j].dest; j++) {
-    if (cattrs[j].val != -1) {
+    if (cattrs[j].val < 0) {
+      XFREE_CLEAR(*cattrs[j].dest);
+      continue;
+    }
+
+    if (cattrs[j].name.type == kObjectTypeString
+        && cattrs[j].name.data.string.size) {
```

Comment:
No need to collapse this, it fits within the 100 char limit

Reply:
Personally I prefer it this way given how long each term is.

As I do most work in splits on a laptop, I usually keep *most* code within 80 chars as a soft limit, with 100 being the hard limit.

If you insist, I don't mind changing it.

Reply:
I understand and appreciate the desire to keep a 80 char limit (I actually pushed against the increase to 100 chars to begin with), but I personally find the two line condition harder to read and parse.

That said this isn't a hill I'm going to die on. It is fine with me -- other folks can chime in if they have an opinion.

Reply:
Ok. I'll change it.

FWIW I think the 100 limit is great for the 5-10% of code that needs it.